contrib/gitian-descriptors/gitian-linux-armhf.yml

```diff
@@ -0,0 +1,124 @@
+---
+name: "bitcoin-linux-armhf-0.13"
+enable_cache: true
+suites:
+- "trusty"
+architectures:
+- "amd64"
+packages:
```

Comment:
If it is not needed, but needed in #7658, it should be *removed* for consistency?